### DISCUSSION TITLE
求解，为什么AndroidJavaObject类Wrap后没有生成带返回值相关的Get方法

### DISCUSSION THREAD

**Discussion by jiaqihuu** [Tencent/puerts#1731] [Q&A]
通过观察UnityEngine_AndroidJavaObject_Wrap发现，M_Call、M_CallStatic，方法都是void类型的，没有重载带返回值的方法；另外也没有重载带返返回值的GetStatic方法。请问是我的导入Puerts方式不对吗，还是因为别的原因，感谢！

**Comment by chexiongsheng** [comment]
返回值的方法是泛型的吧，泛型怎么生成wrapper？枚举世上一切类型作为泛型参数？
这种没约束的泛型，只能通过反射去调用，文档有介绍泛型方法的调用。

**Reply by jiaqihuu** [reply]
![2024-05-23 22 33 22](https://github.com/Tencent/puerts/assets/15024992/6409b9fe-28c2-4a79-bf79-2183ab423701)

收到您的提示后我尝试使用范型和反射的方式调用，但是碰到截图中的问题一直不得解：

控制台报错：invalid arguments to Invoke or the overload is striped by unity，

我怀疑是参数类型不对，但尝试了各种参数还是无法解决。

恳请大佬抽出一点点宝贵时间指导一下下，万分感谢！
